WooCommerce/UITestsFoundation/OrderDataStructs.swift

```diff
@@ -0,0 +1,31 @@
+public struct OrderMock: Codable {
+    public let response: OrderResponseData
+}
+
+public struct OrderResponseData: Codable {
+    public let status: Int
+    public let jsonBody: OrderBodyData
+}
+
+public struct OrderBodyData: Codable {
+    public let data: [OrderData]
+}
+
+public struct OrderData: Codable {
+    public let id: Int
+    public let number: String
+    public var status: String
```

Comment:
Looks like this value doesn't change so can be:
```suggestion
    public let status: String
```